source/extensions/filters/network/zookeeper_proxy/filter.h

```diff
@@ -90,13 +146,16 @@ struct ZooKeeperProxyStats {
   ALL_ZOOKEEPER_PROXY_STATS(GENERATE_COUNTER_STRUCT)
 };
 
+using LatencyThresholdList = const Protobuf::RepeatedPtrField<
+    envoy::extensions::filters::network::zookeeper_proxy::v3::LatencyThreshold>&;
```

Comment:
move `const` and `&` outside of alias and `const LatencyThresholdList&` elsewhere